OpenRA.Mods.Common/Scripting/Properties/GainsExperienceProperties.cs

```diff
@@ -0,0 +1,53 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Scripting;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Scripting
+{
+	[ScriptPropertyGroup("Experience")]
+	public class GainsExperienceProperties : ScriptActorProperties, Requires<GainsExperienceInfo>
+	{
+		readonly GainsExperience exp;
+
+		public GainsExperienceProperties(ScriptContext context, Actor self)
+			: base(context, self)
+		{
+			exp = self.Trait<GainsExperience>();
+		}
+
+		[Desc("The actor's amount of experience.")]
+		public int Experience { get { return exp.Experience; } }
+
+		[Desc("The actor's level.")]
+		public int Level { get { return exp.Level; } }
+
+		[Desc("The actor's maximum possible level.")]
+		public int MaxLevel { get { return exp.MaxLevel; } }
+
+		[Desc("Specifies whether or not the actor can gain a level.")]
```

Comment:
I'm not a native speaker, but to me `specifies` sounds like you'd be able to also `set` this property. `Returns true if the actor can gain a level.` sounds more clear to me. What do you think?

Reply:
I agree. Updated!